test/testinput/bufr_filter_split.yaml

```diff
@@ -50,7 +50,7 @@ observations:
       obsdataout: "./testrun/gdas.t18z.1bmhs.tm00.{splits/hour}.{splits/minute}.filter_split.nc"
 
       dimensions:
-        - name: nchans
+        - name: Chanel 
```

Comment:
typo

Reply:
@jamiebresch Thanks for reviewing this PR.  The type has been corrected and the test output file updated accordingly.